freqtrade/exchange/exchange.py

```diff
@@ -257,6 +259,28 @@ def validate_order_time_in_force(self, order_time_in_force: Dict) -> None:
                 raise OperationalException(
                     f'Time in force policies are not supporetd for  {self.name} yet.')
 
+    def validate_trailing_stoploss(self, config) -> None:
```

Comment:
I really don't think that this should be in the exchange class, but rather in configuration, or in a seperate "validation" file.

it's got nothing to do with the exchange (which is different in the other validate functions ...) ...